WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -214,7 +212,6 @@ public View onCreateView(
     ) {
         mBinding = CommentDetailFragmentBinding.inflate(inflater, container, false);
         mReplyBinding = mBinding.layoutCommentBox;
-        mActionBinding = CommentActionFooterBinding.inflate(inflater, null, false);
```

Comment:
In this file, I removed a lot of logic related to `mActionBinding` which is no longer used.